opts.go

```diff
@@ -14,6 +14,10 @@
 
 package gormetrics
 
+const (
+	DisableGormMetricsDatabaseKey = "gormmetrics-enabled"
```

Comment:
// DisableGormMetricsDatabaseKey can be set on the *gorm.DB object to disable metrics on a particular query

Reply:
added comments you suggested :)